container/id/test/id.go

```diff
@@ -9,15 +9,38 @@ import (
 
 // ID returns random cid.ID.
 func ID() cid.ID {
-	checksum := [sha256.Size]byte{}
+	var res cid.ID
 	//nolint:staticcheck
-	rand.Read(checksum[:])
+	rand.Read(res[:])
+	return res
+}
+
+// NIDs returns n random cid.ID instances.
+func NIDs(n int) []cid.ID {
```

Comment:
not sure if `N` helps of confuses. just `IDs(n int)` is also good imo

Reply:
sure. Actually I started to call them `NType` but `NID` would be unclear. Will rename without `N`

Reply:
done